docs/current_docs/api/secrets.mdx

```diff
@@ -10,11 +10,30 @@ Dagger has first-class support for "secrets", such as passwords, API keys, SSH k
 
 Here is an example, which uses a secret in a Dagger function chain:
 
-<Tabs groupId="language">
-<TabItem value="Dagger CLI">
 ```shell
 export API_TOKEN="guessme"
+```
 
+<Tabs groupId="language">
+<TabItem value="System shell">
+```shell
```

Comment:
@helderco can you please help to get this example working? The system shell version does not work, but the interactive one works.

Reply:
could be the shell is trying to expand `$MY_SECRET`? maybe escape the $

Reply:
try <<'EOF'

Reply:
then you won't need to escape the $

Reply:
> try <<'EOF'

This worked, thank you @shykes !